src/GoogleTranslateConnector.ts

```diff
@@ -0,0 +1,51 @@
+import { Reshuffle, BaseConnector, EventConfiguration } from 'reshuffle-base-connector'
+import TranslationServiceClient, { protos, v3 } from '@google-cloud/translate'
+
+export interface GoogleTranslateConnectorConfigOptions {
+  credentials: string
+}
+
+export default class GoogleTranslateConnector extends BaseConnector<
+  GoogleTranslateConnectorConfigOptions,
+  null
+  > {
+
+  private client: v3.TranslationServiceClient
+
+  constructor(app: Reshuffle, options?: GoogleTranslateConnectorConfigOptions, id?: string) {
```

Comment:
I don't see where you use the credentials from the options. It's not used in v3.TranslationServiceClient()

Reply:
Removed